app/assets/javascripts/student_profile/page_container.jsx

```diff
@@ -1,16 +1,15 @@
 import _ from 'lodash';
 import MixpanelUtils from '../helpers/mixpanel_utils.jsx';
+import PropTypes from '../helpers/prop_types.jsx';
+import * as ReactHelpers from '../helpers/react_helpers.jsx';
+import {fromPair} from '../helpers/from_pair.jsx';
+import {merge} from '../helpers/react_helpers.jsx';
```

Comment:
Looks like line 6 and line 4 are doing the same thing here, since `react_helpers` only exports one function called `merge`. Is this duplicative?